Updates the Gemspec for Wisper 2.0

rringler opened the pull request:
### Description
* Updates the Gemspec for Wisper 2.0.
* Fixes a Typo in the spec directory
* Updates (loosens) the version requirement for the Rubocop gem
* Simplifies `Wisper::Resque` (in-lining the `ResqueBroadcaster` class) for better parity with the `wisper-sidekiq` gem.
* Bumps the `Wisper::Resque::VERSION` constant to '0.2.0'

bzurkowski commented on spec/lib/whisper/resque_spec.rb:
```diff
@@ -0,0 +1,26 @@
+RSpec.describe Wisper do
+  describe Wisper::ResqueBroadcaster do
```
Please, wrap it into single line: 

```ruby
RSpec.describe Wisper::ResqueBroadcaster do
  # ...
end
```

bzurkowski commented on spec/spec_helper.rb:
```diff
@@ -9,17 +9,18 @@
   config.warnings = true
 
-  config.default_formatter = 'doc' if config.files_to_run.one?
+  if config.files_to_run.one?
```
[Rubocop](https://github.com/rubocop-hq/ruby-style-guide#if-as-a-modifier) says: ```Favor modifier if usage when having a single-line body.```

rringler replied:
I was keeping parity with the `wisper-sidekiq` gem, but I don't think that project is enforcing style with rubocop. I will update this.

bzurkowski replied:
I'll setup Travis with Rubocop + RSpec afterwards.

bzurkowski commented on spec/spec_helper.rb:
```diff
@@ -9,17 +9,18 @@
+  end
 
-  config.profile_examples = 10
+  config.profile_examples = 0
```
I'd rather leave the default value.

bzurkowski commented:
Also, please split this two concerns (code simplification and dependency update) into separate  commits or at least modify commit message. Thank you!

rringler committed:
Updates the Gemspec for Wisper 2.0

Updates the Gemspec for Wisper 2.0.

rringler committed:
Fixes Typo in a Spec Directory

Fixes a typo ('whisper' vs. 'wisper') in a spec directory.

rringler committed:
Updates Gemspec to Loosen Rubocop Version

Updates the `.gemspec` file to loosen the Rubocop version specified (locking it
to the minor version instead of the patch version.)

rringler committed:
Updates Class Structure

Updates the Class structure to more closely align with the `wisper-sidekiq` gem.

rringler committed:
Bumps `Wisper::Resque::VERSION` to '0.2.0'

Bumps `Wisper::Resque::VERSION` to '0.2.0'

rringler commented:
Thank you!
